<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="background" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#background{
background-color: qlineargradient(spread:pad, x1:0.323751, y1:0.188, x2:0.727, y2:0.977409, stop:0 rgba(192, 192, 192, 255), stop:1 rgba(255, 255, 255, 255));
}

QLabel#endTitle{
font: 24pt &quot;MS Reference Sans Serif&quot;;
}

QWidget#Container{
background-color: #ffffff;
border-radius: 10px;
}

QLabel#ContainerTitle{
font: 16pt &quot;MS Reference Sans Serif&quot;;
}

QPushButton{
border-radius:5px;
font: 14pt &quot;MS Reference Sans Serif&quot;;
background-color:rgb(205, 124, 255);
}</string>
    </property>
    <widget class="QLabel" name="endTitle_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>631</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Hiragana and Katakana Trainer</string>
     </property>
    </widget>
    <widget class="QWidget" name="Container" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>1101</width>
       <height>501</height>
      </rect>
     </property>
     <widget class="QLabel" name="endTitle">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>381</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Your score is : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="againButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>330</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Again(R)</string>
      </property>
      <property name="shortcut">
       <string>R</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>330</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Quit(Q)</string>
      </property>
      <property name="shortcut">
       <string>Q</string>
      </property>
     </widget>
     <widget class="QLabel" name="finalScore">
      <property name="geometry">
       <rect>
        <x>407</x>
        <y>180</y>
        <width>291</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
